### ISSUE TITLE
Template Artifact Builds

### ISSUE THREAD

**Issue by fbl773** [fbl773/point-b_ank#44]
**Is your feature request related to a problem? Please describe.**
Having to fill in _all_ of the details for a point is going to get really old. It would be nice if I could save a point under a template, then just assign that spec to a point/points.

**Describe the solution you'd like**
If we could have an option on a specific point to the effect of `save as template` that is one way this could be done. Then users could save that template to the system for everyone to use. 

**Describe alternatives you've considered**
Originally I thought about tying it to a specific culture... And we probably should do this _somehow_ seeing as a "culture" is essentially a template for a point's attributes. Perhaps this is where the templates could _live_ and be managed from. As part of the _culture_ page. This is a distant feature, should meet on it when the time comes.

**Comment by Flake-bender** [comment]
> **Is your feature request related to a problem? Please describe.** Having to fill in _all_ of the details for a point is going to get really old. It would be nice if I could save a point under a template, then just assign that spec to a point/points.
> 
> **Describe the solution you'd like** If we could have an option on a specific point to the effect of `save as template` that is one way this could be done. Then users could save that template to the system for everyone to use.
> 
> **Describe alternatives you've considered** Originally I thought about tying it to a specific culture... And we probably should do this _somehow_ seeing as a "culture" is essentially a template for a point's attributes. Perhaps this is where the templates could _live_ and be managed from. As part of the _culture_ page. This is a distant feature, should meet on it when the time comes.

Yes, agreed. Culture/point types should be saved templates, and all the main types found in Sask, with the appropriate age/date ranges should be pre-saved as templates for our users. Also, I'm still hoping to get some line drawings of the main types, to add to the templates, to help folks recognize which they have.
